Integrations/integration-IntSights.yml

```diff
@@ -498,7 +522,8 @@ script:
             'Flags': {'IsInAlexa': demisto.get(r, 'Flags.IsInAlexa')},
             'Enrichment': {
                   'Status': demisto.get(r, 'Enrichment.Status'),
-                  'Data': demisto.get(r, 'Enrichment.Data')
+                  'Data': demisto.get(r, 'Enrichment.Data'),
+                  'Date': demisto.get(r, 'Enrichment.Data') # Backwards compatability issue
```

Comment:
Did we remove `Date` from the outputs?

Reply:
It was never in the outputs, but i know that Guy searched for `Date` in his script so other customers might have done that as well